shared/validation-schemas/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@animavita/validation-schemas",
```

Comment:
Why this need to be shared?

Reply:
Cause then we validate on the client as well matching the same schema.